src/layout/header/Header.css

```diff
@@ -9,6 +9,16 @@
   background: #303230;
 }
 
+@media (width <= 750px) {
```

Comment:
great, thanks. let's just be consistent with the mobile width
https://github.com/hasadna/open-bus-map-search/blob/f924722ad6e55d0b8b846c0a40eaa449d32c81d4/src/layout/sidebar/sidebar.scss#L17

Reply:
Done